gtests/net/mptcp/add_addr/add_addr6_client.pkt

```diff
@@ -0,0 +1,33 @@
+// connection initiated by the kernel
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0     `../common/client.sh`
+
++0.0   socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0   setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0   fcntl(3, F_GETFL) = 0x2 (flags O_RDWR)
++0.0   fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
++0.0   connect(3, ..., ...) = -1 EINPROGRESS (Operation now in progress)
++0.0      >  S   0:0(0)                    <mss 1460, sackOK, TS val 100 ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0.0      <  S.  0:0(0)  ack 1  win 65535  <mss 1460, sackOK, TS val 700 ecr 100, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
++0.0      >   .  1:1(0)  ack 1             <nop, nop,         TS val 100 ecr 700,                mpcapable v1 flags[flag_h] key[ckey, skey]>
+
++0.200  getsockopt(3, SOL_SOCKET, SO_ERROR, [0], [4]) = 0
++0.205  fcntl(3, F_SETFL, O_RDWR) = 0   // set back to blocking
+
++0.01   write(3, ..., 100) = 100
++0.0      >  P.  1:101(100)  ack 1         <nop, nop, TS val 305 ecr 700, mpcapable v1 flags[flag_h] key[ckey, skey] mpcdatalen 100, nop, nop>
+
+// explicit ACK then ADD_ADDR: on some slow env, injecting ADD_ADDR can take
+// time, causing the host to retransmit its last data packet. Here we ensure
+// this previous packet is acked then we can send the ADD_ADDR
++0.0      <   .  1:1(0)  ack 101  win 256  <nop, nop, TS val 705 ecr 305,                                    dss dack8=101 dll=0 nocs>
```

Comment:
same here, not sure why the `dss` is here. Same at line 33 :)

Reply:
ok